source/community/reactnative/README.md

```diff
@@ -65,6 +69,11 @@ Registry.getManager().registerComponent('Rating',RatingComponent);
 Registry.getManager().removeComponent('Input.Date');
 ```
 
+* To override an internal the support of an element type (i.e To ignore the rendering of specific element type), 
```

Comment:
How this is different from the above line? It looks repeating for me.

Reply:
resolved